submissions/ayodeleayobami/landingpage/task.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```

Comment:
Your indentation is off here, please refer to the html section in the STYLEGUIDE.md to fix.

Reply:
done